src/Geopilot.Frontend/public/locale/de/common.json

```diff
@@ -0,0 +1,57 @@
+{
+  "about": "Über",
+  "appSubTitle": "Online Validierung & Lieferung von Geodaten",
+  "bugTracking": "Bugtracking",
+  "cancel": "Abbrechen",
+  "close": "Schliessen",
+  "codeLicenseInfo": "Der Code steht unter der <licenseLink>MIT Lizenz</licenseLink> im <repositoryLink>Github Repository</repositoryLink> zur Verfügung. Falls Ihnen Bugs begegnen, können Sie dort ein <issuesLink>Issue</issuesLink> eröffnen.",
+  "comment": "Kommentar",
+  "createDelivery": "Lieferung erstellen",
+  "createDeliverySuccess": "Lieferung erfolgreich",
+  "createDeliveryFailed": "Bei der Abgabe ist ein unbekannter Fehler aufgetreten, bitte versuchen Sie es mit einer neuen Validierung.",
+  "delete": "Löschen",
+  "deleteDelivery_one": "{{count}} Datenlieferung löschen",
+  "deleteDelivery_other": "{{count}} Datenlieferungen löschen",
+  "deleteDeliveryConfirmation": "Möchten Sie die Datenabgabe wirklich löschen? Diese Aktion kann nicht rückgängig gemacht werden.",
```

Comment:
Q: Hier wird die Höflichkeitsform verwendet, ist das korrekt? In anderen Texten verwenden wir das _Du_ (z.B. Drop-Zone).

Reply:
@tschumpr Ich würde ebenfalls auf die Höflichkeitsform setzten, da die Akteure miteinander kommunizieren.